obs2ioda-v2/src/ncio_mod.f90

```diff
@@ -1,17 +1,19 @@
 module ncio_mod
 
-use netcdf
-use kinds, only: i_kind, r_single, r_kind
-use define_mod, only: nobtype, nvar_info, n_ncdim, n_ncgrp, nstring, ndatetime, &
-   obtype_list, name_ncdim, name_ncgrp, name_var_met, name_var_info, name_sen_info, &
-   xdata, itrue, ifalse, vflag, ninst, inst_list, write_nc_conv, write_nc_radiance, &
-   write_nc_radiance_geo, ninst_geo, geoinst_list, &
-   var_tb, nsen_info, type_var_info, type_sen_info, dim_var_info, dim_sen_info, &
-   unit_var_met, iflag_conv, iflag_radiance, set_brit_obserr, set_ahi_obserr
-use netcdf_mod, only: open_netcdf_for_write, close_netcdf, &
-   def_netcdf_dims, def_netcdf_grp, def_netcdf_var, def_netcdf_end, &
-   put_netcdf_var, get_netcdf_dims
-use ufo_vars_mod, only: ufo_vars_getindex
+use kinds,only : i_kind,r_single,r_kind,i_llong
```

Comment:
In my opinion, these changes to whitespace make it really difficult to review the PR. Is there any way to construct commits that only include changes necessary to use the netCDF C++ wrappers in the ncio_mod module?